packages/wallets/src/components/Base/IconButton/IconButton.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { ComponentProps, CSSProperties, FC } from 'react';
+import classNames from 'classnames';
+import { TGenericSizes } from '../types';
+import './IconButton.scss';
+
+interface IconButtonProps {
+    color?: CSSProperties['color'] | 'primary' | 'transparent';
```

Comment:
Isn't `'transparent'` included in `CSSProperties['color']`?

Reply:
Thanks for pointing that out @rostislav-deriv will make the changes accordingly 🙏🏼